spec/requests/users/confirmations_spec.rb

```diff
@@ -4,16 +4,38 @@
 
 RSpec.describe "Users::Confirmations", type: :request do
   let(:confirmation_token) { "soC7rF8i_BgYdUxafFcP" }
+  let(:school) { create(:school) }
+  let(:induction_coordinator) { user.induction_coordinator_profile }
 
   let!(:user) do
-    create(:user, confirmed_at: nil, confirmation_token: confirmation_token)
+    create(:user, :induction_coordinator, confirmed_at: nil, confirmation_token: confirmation_token)
   end
```

Comment:
We're going to have users who aren't induction coordinators...

Reply:
We are not sure at this point but I agree I need to restrict the notification to induction coordinator confirmations